.github/workflows/ci.yaml

```diff
@@ -63,6 +63,12 @@ jobs:
           # TODO: put back zfpy import when it supports numpy 2.0
           python -m pip install -v -e .[test,test_extras,msgpack,pcodec]
 
+      - name: Install zarr-python
+        shell: "bash -l {0}"
+        run: |
+          conda activate env
+          python -m pip install zarr==3.0.0a0
```

Comment:
We need a way to run tests on both zarr v2 and zarr v3 before this is merged.

Reply:
Just had another look. The `zarr` package is only used for these Zarr3 codecs and their corresponding tests. Previously, `zarr` wasn't even installed in the CI. I don't think we need to test against zarr-python 2, here.

Reply:
Ahhh that makes sense, don't know why I got confused!